release-post.md

```diff
@@ -0,0 +1,68 @@
+# Universal Sierra Compiler
+
+The Software Mansion Team has created a new tool, Universal Sierra Compiler (further abbreviated as USC).
+
+USC is the tool for Sierra compilation - it compiles any ever-existing Sierra version to CASM.
+
+The idea behind USC is simple, yet it might make things easier for some tools.
+It bundles all Sierra compilers previously released, and depending on the version contained inside the Sierra JSON, it chooses the right compiler.
+Therefore it allows any tool to be independent of Sierra version.
+
+Universal Sierra Compiler might become especially useful for nodes, which have to interact with contracts deployed in different versions of Sierra, but also any other tool that depends on the versioning of Sierra.
+
+
+## Usage
+
+| Command          | Purpose                               | Input                        | Input as JSON of Cairo struct                              | Output as JSON of Cairo struct |
+|------------------|---------------------------------------|------------------------------|------------------------------------------------------------|-|
+| compile-contract | compile Sierra of a Starknet contract | output of `cairo-compile`    | cairo_lang_starknet_classes::contract_class::ContractClass | cairo_lang_starknet::casm_contract_class::CasmContractClass |
+| compile-raw      | compile Sierra of a Cairo program     | output of `starknet-compile` | cairo_lang_sierra::program::Program                        | |
+|                  |                                       |                              |                                                            | |
+
+
+```shell
+$ universal-sierra-compiler compile-raw --sierra-path ./path/to/sierra.json
+
+{"assembled_cairo_program": ...}
+```
+
+
+## Installation
+
+Use the provided script to install the latest version of Universal Sierra Compiler:
+
+```shell
+curl -L https://raw.githubusercontent.com/software-mansion/universal-sierra-compiler/master/scripts/install.sh | sh
+
+...
+universal-sierra-compiler (v1.0.0) has been installed successfully.
+```
+
+It is also possible to install a specific version of USC by passing, for example:
+
+```shell
+| sh -s v2.0.0-rc0
```

Comment:
Somebody will copy this and ask why it's not working

Reply:
![image](https://github.com/software-mansion/universal-sierra-compiler/assets/65187002/783f9048-c99a-42b2-9c1f-0bf33eb0b12f)